packages/account/src/Components/file-uploader-container/file-uploader-container.tsx

```diff
@@ -1,59 +1,29 @@
 import React from 'react';
-import classNames from 'classnames';
-import { Icon, Text } from '@deriv/components';
-import { PlatformContext, isDesktop, WS, TSettings } from '@deriv/shared';
-import { Localize, localize } from '@deriv/translations';
+import { Text } from '@deriv/components';
+import { isMobile, WS } from '@deriv/shared';
+import type { TSettings } from '@deriv/shared/src/utils/files/file-uploader-utils';
```

Comment:
Why is this split from `@deriv/shared` ?

Reply:
@likhith-deriv, I faced this error `Cannot use namespace 'TSettings' as a type.` while building the project